packages/my-queries-storage/src/compass-query-storage.ts

```diff
@@ -74,8 +104,8 @@ export class CompassRecentQueryStorage
       const lastRecent = recentQueries[recentQueries.length - 1];
       await this.delete(lastRecent._id);
     }
-
-    const _id = new UUID().toString();
+    // TODO: verify that this doesn't break anything in compass
+    const _id = new ObjectId().toHexString();
```

Comment:
Was there something wrong with UUIDs?

Reply:
In either case we haven't changed the type, just the string format, is that what we want here? just making sure if we want an ObjectId or UUID to be stored, we shouldn't toString this, we should leave it as the type we want